site/en/docs/privacy-sandbox/chrome-testing/index.md

```diff
@@ -6,138 +6,184 @@ subhead: >
+### Accessing the Sec-Cookie-Deprecation HTTP header {: #cookie-deprecation-header}
+
+To receive the Sec-`Cookie-Deprecation` request header, a site must first set the `receive-cookie-deprecation` cookie. This cookie must use the [`Partitioned` attribute](/docs/privacy-sandbox/chips/), which means that opt-in for receiving the header must be done per top-level site.
 
 For example, if `3p-example.site` wants to receive the `Sec-Cookie-Deprecation` header on its resources embedded on `example.com`, then `3p-example.site` must set the following cookie in that context.
 
 ```text
 Set-Cookie: receive-cookie-deprecation=1; Secure; HttpOnly; Path=/; SameSite=None; Partitioned;
 ```
 
-The `Secure`, `HttpOnly`, `Path`, `SameSite`, and `Partitioned` cookie attributes are mandatory. The other attributes: `Domain`, `Expires`, and `Max-Age` may be set as best suits your needs.
+The `Secure`, `HttpOnly`, `SameSite`, and `Partitioned` cookie attributes are mandatory. The other attributes: `Domain`, Path (though Path=/ is a good default), `Expires`, and `Max-Age` may be set as best suits your needs.
 
 Assuming the browser was in the `example_label_1 group`, subsequent requests which include this cookie would also include the `Sec-Cookie-Deprecation` header.
 
+
 ```text
 Sec-Cookie-Deprecation: example_label_1
 ```
 
 If the browser is not part of a group, no header will be sent.
 
-As labels are tied to the presence of the cookie, if the cookie is deleted, labels will also no longer be sent. As the `Partitioned` attribute is intended for continued use after third-party cookies are fully deprecated, this means `Partitioned` cookies may be set when third-party cookies are blocked.
+Labels are tied to the presence of the cookie, so if the cookie is deleted, blocked entirely, or blocked for the specific site, then labels will not be sent. As the `Partitioned` attribute is intended for continued use after third-party cookies are fully deprecated, this means `Partitioned` cookies may be set when third-party cookies are blocked.
 
 
-### Accessing the cookieDeprecationLabel JavaScript API
+### Accessing the cookieDeprecationLabel JavaScript API {: #cookie-deprecation-js}
 
-The `Cookie-Deprecation` value can also be accessed via the `navigator.cookieDeprecationLabel.getValue()` JavaScript API. This will return a promise which resolves to a string containing the applicable group label. For example, if the browser was in the `example_label_1` group:
+The `Cookie-Deprecation` value can also be accessed via the `navigator.cookieDeprecationLabel.getValue()` JavaScript API. This will return a promise which resolves to a string containing the applicable group label. For example, if the browser was in the `example_label_1 `group:
```

Comment:
```suggestion
The `Cookie-Deprecation` value can also be accessed via the `navigator.cookieDeprecationLabel.getValue()` JavaScript API. This will return a promise which resolves to a string containing the applicable group label. For example, if the browser was in the `example_label_1`group:
```